### ISSUE TITLE
Installation guide is only meant to be run from main branch?

### ISSUE THREAD

**Issue by ajoga** [ail-project/lacus#19]
Hello,

I'm following the guide to install this, but with a twist as I checkout the latest tag in order to not run from the main branch. Is this unsupported and shouldn't be done, or wasn't considered?

I do my best to run known releases of things, but let me know if this shouldn't be done here.

OS: Ubuntu 22.04.4 LTS

Steps to reproduce:

1/ install AIL
2/ 

```bash
$ pwd # ensure you're in the AIL directory
$ sudo apt install pipx
$ pipx install poetry # recommended way to install poetry, by poetry: https://python-poetry.org/docs/
$ pipx ensurepath
$ git clone https://github.com/ail-project/lacus.git
$ cd lacus
$ git checkout v1.10.0 # latest release at time of writing this
$ poetry install
$ poetry shell
$ playwright install-deps
$ echo LACUS_HOME="`pwd`" >> .env
$ poetry run update --init
No logging config file available.
Falling back on sample config, please initialize the logging config file.
* Update repository.


You are not currently on a branch.
Please specify which branch you want to merge with.
See git-pull(1) for details.

    git pull <remote> <branch>
```

I think it comes from https://github.com/ail-project/lacus/blob/2482a07e59d46517f57efcbe259f2c93afe7c568/bin/update.py#L72 

Thank you

**Comment by Rafiot** [comment]
It's just not been considered. I'm in the middle of something else right now, so I won´t be able to implement a way to optionally track the tag instead of the main branch only today, but that would be a good addition.

**Comment by tarbo03** [comment]
Hi, I have followed the step by step to install the AIL Framework, I have installed the Lacus as well but not working out for me as am getting error message about Crawler Disabled, Screenshot below:
![Image](https://github.com/user-attachments/assets/da3e8a56-8142-4216-b7bd-f15394965f12)

Below is screenshots as I have installed, take a look at them maybe gonna be helpful.

![Image](https://github.com/user-attachments/assets/2ce169c9-829d-4a9c-bb4c-4d726894a344)

![Image](https://github.com/user-attachments/assets/15f12741-db75-4404-8419-d2da7bb21380)

I need your support on this.
Thanks very much in advance.

**Comment by tarbo03** [comment]
Thanks very much @Terrtia , I really appreciate your support, I have added the lacus url in the AIL settings just as you have instructed AIL Lacus crawler shows "Connected" but the Crawler still throws error message. Check out the screenshot below for the error.

![Image](https://github.com/user-attachments/assets/5b476e06-67f5-41f4-b0e3-e65fd09b2154)

Thanks once again and regards.

**Comment by tarbo03** [comment]
Good day, I am still having a hard time resolving the crawler issue, I sincerely need support @Terrtia 
Thanks very much and I look forward to hearing from you son.

**Comment by tarbo03** [comment]
Hello @Rafiot , Thanks very much for your support I have been battling with the installation guide for Lacus but seems not to be working for me, I am getting this error messages about "package has no installation candidate"  "Package is not available, but is referred to by another package"

I have attached screenshots below for better understanding of the problem.

![Image](https://github.com/user-attachments/assets/a9f6ea35-eb65-4094-89c9-f2e1021529f1)
![Image](https://github.com/user-attachments/assets/0e0188de-c3f3-4d47-87b7-eae127393321)

I am tryna install on my Kali Linux machine.

Thanks and regards.

**Comment by Rafiot** [comment]
Sadly, Kali isn't supported by Playwright (https://playwright.dev/docs/intro#system-requirements), so it won't work, and I do not know how to fix it.

You should use Ubuntu to run lacus.

**Comment by tarbo03** [comment]
Ok thanks very much @Rafiot  I really do appreciate your support, am gonna try installing Ubuntu and go through the Playwrite installation process again.
Thanks and regards.

**Comment by tarbo03** [comment]
Hello @Rafiot , I was able to migrated over to Ubuntu, tryna install poetry but getting this error in the screenshot bellow:

![Image](https://github.com/user-attachments/assets/c3e4c83c-ab4d-48f6-8c06-696b6857c981)

May you can help.
Thanks.

**Comment by Rafiot** [comment]
How did you install Lacus, and did you do it this way: https://github.com/ail-project/lacus?tab=readme-ov-file#system-dependencies

**Comment by tarbo03** [comment]
Hell @Rafiot  Thank you once again, I installed Lacus following the instructions above I got few error during the "make test" find screenshot below for better understanding.

![Image](https://github.com/user-attachments/assets/1af34bed-f7b6-4aa1-a800-6f27ae5f6fd1)

Thanks and regards

**Comment by Rafiot** [comment]
I don't know, but it seem that this test is testing the latency, so it's possibly because the machine you're using is too slow for that test? 

It may not be a big deal, but it might also mean that lacus will be extremely slow.

**Comment by tarbo03** [comment]
Ok, I have tried defragmenting the VM disk, am gonna try test again and see maybe issue will be resolved but if not resolve am just gonna ignore the error and proceed. am gonna update you on the outcome

**Comment by Rafiot** [comment]
I doubt defragmenting will do anything. 

Just making sure you know: Lacus spins up a full browser (headless, but full) to capture pages. If the machine is extremely small, it may not work great.

**Comment by tarbo03** [comment]
When I launched the poetry see bellow the screenshot below:

![Image](https://github.com/user-attachments/assets/04f08b69-739a-4f30-9628-a8b330d1852f)

![Image](https://github.com/user-attachments/assets/73c23f47-18ff-43bd-9bd5-10cc371140af)

Maybe its gonna explain better.

**Comment by Rafiot** [comment]
Right, seems it's not documented. You need to also install tor on the machine so lacus can capture onion URls.

`apt install tor` should do the trick

**Comment by tarbo03** [comment]
Ok, am gonna install the tor and try again, will give you feedback accordingly.
Do I also have to start the tor service each time I want to power up the AIL Framework?

**Comment by tarbo03** [comment]
Success!!! It worked now all connected, see screenshot bellow 

![Image](https://github.com/user-attachments/assets/39404dd6-0d2b-4fcd-9d22-7087a282ae6e)

**Comment by tarbo03** [comment]
But I still havent start getting feeds yet, look at the lacus API Configuration bellow is there anything other thing I need to do to make it right?

![Image](https://github.com/user-attachments/assets/992bdf3e-30b0-4676-b503-011f735ca0d5)

![Image](https://github.com/user-attachments/assets/c4efade9-f8d7-4747-bb09-4b9adb6d2668)

**Comment by tarbo03** [comment]
Hello @Rafiot @Terrtia  I have not start getting feeds yet? I need support on this, thanks very much for my progress so far.

**Comment by Rafiot** [comment]
You need to setup your own feed(s): https://github.com/ail-project?q=feed&type=all&language=&sort=

**Comment by tarbo03** [comment]
Ok thanks very much @Rafiot . I got one question, to setup my own feed for example this feed https://github.com/ail-project/ail-feeder-leak do I need to clone this into the same directory with the AIL or inside the AIL directory?
Thanks and regards.

**Comment by tarbo03** [comment]
Good day @Rafiot @Terrtia  I have been tryna setup feeder-leak but getting this error messages below while installing Requirements.txt 

![Image](https://github.com/user-attachments/assets/359b3d8f-6d0a-45ed-b3dd-780966255ece)

**Comment by tarbo03** [comment]
Hello @Rafiot @Terrtia @adulau  I have manually crawled a few onion domains and got some feeds as you can see in the below screenshot: 

![Image](https://github.com/user-attachments/assets/eecf0cf3-d007-4245-b4a7-a3d0bfb66794)


I would like to get feeds from Telegram I have follow the documentation of it but seems not to be clear to me, I need support.

Thanks and regards.

**Comment by Rafiot** [comment]
I'd like to point out that you're being extremely demanding and we are not at your service.

The complete toolchain lacks in documentation but it is also complicated for us to figure out exactly what you did and how you end-up in the situations you're in with just screenshots. And as far as I can tell, lacus is working for you now, so please open issues in the relevant projects. This one was left open as a note for me to implement a way to update to tagged releases, not for generic support questions. 

Closing it now, and opening a dedicated note for myself in another issue.

**Comment by tarbo03** [comment]
Thanks very much @Rafiot, I really appreciate all your support and also I totally understand you, I had recommended the AIL Framework to be used in our CERT operation, the who management team seems to be interested in it and needed a prove of concept or lemme say like a demonstration to see how the framework works.

**Comment by Terrtia** [comment]
Hi @tarbo03 !

I updated the AIL Documentation: https://github.com/ail-project/ail-framework/tree/master/doc
Let me know if something is missing (please open an issue [here](https://github.com/ail-project/ail-framework/issues))
